### ISSUE TITLE
I have another question about training，can you help me see it，thanks!

### ISSUE THREAD

**Issue by zengbing1992** [mozilla/LPCNet#131]
The u8 and f32 files generated by training, where u8 is a pcm file, then my question is whether the pcm file must be in u8 format? My understanding is to use the first three saved data of the u8 file for training, and then use the u8 file Is the fourth data calculate loss to adjust the weight of the model? I don’t know if my understanding is correct?

**Comment by yingfenging** [comment]
Yes,your understanding is correct, the u8 file must be u8 format, because the loss function is sparse_categorical_crossentropy ,

**Comment by zengbing1992** [comment]
@yingfenging thank you so much!

**Comment by justforhome** [comment]
@yingfenging Did you train the  LPCNet using the Chinese dataset ? Could you please tell me your email address ?

**Comment by yingfenging** [comment]
Please leave your email， and I will connect you.

> @yingfenging Did you train the LPCNet using the Chinese dataset ? Could you please tell me your email address ?

**Comment by wangshiyao-1119** [comment]
> Please leave your email， and I will connect you.
> 
> > @yingfenging Did you train the LPCNet using the Chinese dataset ? Could you please tell me your email address ?

Hello, I also want to use the Chinese dataset to train, can I discuss it with you? This is my email address：1067701367@qq.com
